libs/financial-aid/shared/src/lib/formatters.ts

```diff
@@ -32,8 +32,13 @@ export const getState: KeyMapping<ApplicationState, string> = {
 export const insertAt = (str: string, sub: string, pos: number) =>
   `${str.slice(0, pos)}${sub}${str.slice(pos)}`
 
-export const formatPhoneNumber = (phoneNumber: string) =>
-  insertAt(phoneNumber.replace('-', ''), '-', 3) || '-'
+export const formatPhoneNumber = (phoneNumber: string) => {
```

Comment:
Not against this function, just asking, does the api return the phonenumber in different formats?

Reply:
I was not sure what format the authentication would return, so I made this just in case. However after testing from looks like the phone number is formatted in +3540000000, should we just always make assumption that it will look like that?